tox.ini

```diff
@@ -935,11 +935,21 @@ passenv = HOME, USER
 setenv=
     CLUSTER_NAME = {env:CLUSTER_NAME:test-enclave-1}
     CLUSTER_HTTP_PORT={env:CLUSTER_HTTP_PORT:9083}
-    DEVSPACE_PROFILE=enclave
+    K3S_DOCKER_IMAGE={env:K3S_DOCKER_IMAGE:docker.io/rasswanth/k3s:v1.28.8-k3s1-cuda-12.2.0-base-ubuntu22.04}
+    CLUSTER_ARGS={env:CLUSTER_ARGS:--volume /sys/kernel/security:/sys/kernel/security --volume /dev/tmprm0:/dev/tmprm0}
+    DEVSPACE_PROFILE={env:DEVSPACE_PROFILE:enclave-cpu}
 allowlist_externals =
     tox
+    bash
 commands =
-    tox -e dev.k8s.start -- --volume /sys/kernel/security:/sys/kernel/security --volume /dev/tmprm0:/dev/tmprm0
+    bash -c "echo k3S_DOCKER_IMAGE=$K3S_DOCKER_IMAGE GPU_ENABLED=$GPU_ENABLED"
+    bash -c "if [ $DEVSPACE_PROFILE == 'enclave-gpu' ]; then \
+      echo 'Starting cluster with GPU support'; \
```

Comment:
[PENDING]: Move this to GPU initialisation for different server types domain, enclave, gateway